src/cim/components/helpers/toAssisted.ts

```diff
@@ -48,23 +43,14 @@ export const getAIHosts = (
         createdAt: agent.metadata?.creationTimestamp,
         validationsInfo: JSON.stringify({ hardware: [] }),
         inventory: JSON.stringify(inventory),
-        progress: {
-          currentStage,
-          progressInfo,
-          stageStartedAt: stageStartTime,
-          stageUpdatedAt: stageUpdateTime,
+        progress: agentProgress && {
```

Comment:
This can result in `false` boolean value. Is it allowed value for the `progress`? What about using `undefined` when `!agentProgress`?

Reply:
hmm, host.progress  `agentProgress` is `HostProgress | undefined`. I don't see how the result could be `false`